hoot-core-test/src/test/cpp/hoot/core/util/MapProjectorTest.cpp

```diff
@@ -68,19 +66,13 @@ class MapProjectorTest : public CppUnit::TestFixture
   boost::minstd_rand rng;
   ofstream fs;
 
-  void setUp()
+  virtual void setUp()
   {
-    TestUtils::resetEnvironment();
+    HootTestFixture::setUp();
     // we are testing the map reprojector so we don't want to force to a single projection.
     conf().set(ConfigOptions::getTestForceOrthographicProjectionKey(), false);
   }
 
-  void tearDown()
-  {
-    TestUtils::resetEnvironment();
-  }
-
-
```

Comment:
Some classes already had the `tearDown` function that does exactly what the new `tearDown` function does so I deleted it.